.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Build ${{ env.PACKAGE_NAME }}
         run: |
           echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u awslabs --password-stdin
-          export DOCKER_IMAGE=ghcr.io/awslabs/aws-iot-device-client/ubuntu-16-x64:latest
+          export DOCKER_IMAGE=ghcr.io/awslabs/aws-iot-device-client/ubuntu-18-x64:latest
```

Comment:
* You should rename the label (line 43) from `build-ubuntu-16-x64` to `build-ubuntu-18-x64`.  
* Also I think we have to replace all instances in this file `s/ubuntu-16-x64/ubuntu-18-x64/g` since we are essentially bumping our ubuntu image from 16 to 18.